app/konnect/service-catalog/index.md

```diff
@@ -4,17 +4,127 @@ subtitle: Track every service across your architecture
 content-type: explanation
 ---
 
-A new {{site.service_catalog_name}} experience is currently in development. The new {{site.service_catalog_name}} will be a one-stop place to see all your services, with all your service information, right at your fingertips. 
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization. This catalog gives you insight into your organization's services and their dependencies. It includes both {{site.base_gateway}} services and external integration resources.
```

Comment:
Suggestion: Kong Konnect's Service Catalog offers a comprehensive catalog of all services running in your organization. By integrating with both Konnect-internal applications (i.e. Gateway Manager, Mesh Manager), as well as external applications (i.e. GitHub, PagerDuty), Service Catalog provides you with a 360 overview into each of your organization's services. It presents you with information that include who the service's owner is, its upstream/downstream dependencies, its code repositories, its CI/CD pipelines, and whether it is fronted by an API gateway or is part of a service mesh.